bandwidth-file-spec.txt

```diff
@@ -128,6 +128,9 @@ It consists of:
     If there are scanners that do not run continuously, they SHOULD be
     excluded from the timestamp calculation.
 
+    If a generator does not have information about the most recent result,
+    it MUST NOT generate the file.
```

Comment:
This doesn't make sense. And it is incomplete.

If there is no recent result, the generator MUST delete the old file, and it MUST NOT generate a new file.

Reply:
After our discussion on trac, this is complete, but it does not make sense.

If there are no recent results, the generator MUST NOT generate a new file.